src/status_im/common/alert_banner/view.cljs

```diff
@@ -0,0 +1,57 @@
+(ns status-im.common.alert-banner.view
+  (:require [quo.core :as quo]
+            [quo.foundations.colors :as colors]
+            [react-native.core :as rn]
+            [react-native.hole-view :as hole-view]
+            [react-native.safe-area :as safe-area]
+            [status-im.common.alert-banner.style :as style]
+            [utils.re-frame :as rf]))
+
+(def colors-map
+  {:alert
+   {:background-color colors/warning-50-opa-20
```

Comment:
or if theme can be passed even better as I believe theme should always be dark passed to the warning banners?
Probably we can wrap these banners in a theme-provider with dark theme applied 👍

Reply:
`{:background-color (colors/resolve-color :warning **:light** 20)` ?

Reply:
ok, when opacity, always light.

Reply:
nope if you look at the colors system - it is always using the "50" value when there is opacity, regardless of theme - this is the way design have the colors system working at the moment. It's one of the reasons it's better to do it this way, so it will be easier to adjust the system should they adjust it going forward.